gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/anonymous/AnonymousConfig.groovy

```diff
@@ -35,15 +35,14 @@ import org.springframework.security.web.authentication.AnonymousAuthenticationFi
 @ConditionalOnExpression('${auth.anonymous.enabled:false}')
 @Configuration
 @ConfigurationProperties(prefix = "auth.anonymous")
-class AnonymousConfig implements AuthConfig.WebSecurityAugmentor {
+class AnonymousConfig {
```

Comment:
Nope. The anonymous config will need to be reworked, just like the other ones. Want to take a stab at it?